### PR TITLE
Fix relation dragline over services

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3682/
Test PASSed.

**Comment by makyo** [comment]
# QA

1. Dragging over unrelatable services
  * add mariadb and hadoop
  * try and relate (both via add rel button and long click)
  * when dragging over the service, the line should follow the cursor
2. Dragging over related services
  * add mariadb and wordpress
  * ensure that the drag line snaps into and out of the service (both via add rel button and long click)
  * pay special attention to where the corners of the service block used to be (we switched from a pointInside method that worked with rects to a pointInside method that works with circles)
3. Pan/zoom obeyed
  * Ensure that both of the above work at various pan/zoom levels

File: jujugui/static/gui/src/app/views/topology/service.js [hunk 1]
```diff
@@ -650,12 +650,6 @@ YUI.add('juju-topology-service', function(Y) {
 
       // Do not fire unless we're within the service box.
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/topology/service.js]: Looks like this comment could be removed now.

**Comment by huwshimi** [comment]
Nice work @makyo so much better. QA OK :+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3683/
Test PASSed.

**Comment by hatched** [comment]
code :+1: Thanks for figuring this out!

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**